### DISCUSSION TITLE
Skin Distance Parameter

### DISCUSSION THREAD

**Discussion by ianibis** [simnibs/simnibs#348] [8. Simulation workflows]
Hello,

I am modeling electric fields for TMS with SimNIBS and I have a question about a parameter.

Is the "Skin Distance" field in SimNIBS representative of the distance between the scalp and the coil or the scalp and the coil casing? Also, if it is the distance between the scalp and the coil, is the measurement from the center or outer edge of the coil?

After reviewing the SimNIBS website, the SimNIBS tutorial article, and the article on the 25 validated coil models, I could not determine the answer. Thank you for your time!

**Comment by TorgeW** [comment]
The Skin Distance for a fig of 8 coil is the distance between the coil casing surface at the center of the coil and the scalp. 
For some other coils the zero position might be at a different location (not on the casing surface) so that the coil sits on top of the head when the skin distance is set to 0.

**Reply by ianibis** [reply]
Just to confirm, by "center of the coil," do you mean the center of the figure-8 shape of the coil or the point between the handle and the figure-8 shape? I'm wanting to confirm because it appears that the target location from SimNIBS correlates with the latter on gmsh when I display the head shape and the coil. Moreover, the centroid appears to be at the point between the handle and the figure-8 shape when I model the coil
<img width="960" alt="image3" src="https://github.com/user-attachments/assets/829d4fe2-ef9d-4cc6-b2d0-8e1983de6069">
<img width="480" alt="image2" src="https://github.com/user-attachments/assets/51e142ed-47b4-46bd-9328-a11d0dba7532">
<img width="480" alt="image1" src="https://github.com/user-attachments/assets/ecdc5ee9-5f3b-4a97-8f1f-9f5486aeb981">

**Reply by TorgeW** [reply]
The center of the coil is the coordinate center.
You can see it marked by the red sphere in this image.

![image](https://github.com/user-attachments/assets/d8ae859d-b844-4da8-9b2b-e3a9935a2188)
